launcher/ui/MainWindow.ui

```diff
@@ -779,8 +738,7 @@
   </action>
   <action name="actionViewCatPackFolder">
    <property name="icon">
-    <iconset theme="viewfolder">
-     <normaloff>.</normaloff>.</iconset>
```

Comment:
The QT-Editor removed the `<normaloff>` tag for every icon. Is that okay or should I try to undo?